modules/govuk_jenkins/templates/jobs/deploy_app_downstream.yaml.erb

```diff
@@ -45,8 +45,8 @@
           GITHUB_AUTH="-H 'Authorization: token $GITHUB_API_TOKEN'"
```

Comment:
You can delete this line too.

Reply:
Oops, I changed approach halfway through. Done.